pkg/log/log_test.go

```diff
@@ -128,7 +128,7 @@ var _ = Describe("logging", func() {
 		BeforeEach(func() {
 			root = &fakeLoggerRoot{}
 			baseLog = &fakeLogger{root: root}
-			delegLog = NewDelegatingLogger(NullLogger{})
+			delegLog = NewDelegatingLogger(DefaultNullLogger())
```

Comment:
This seems kind of a breaking change if we now require to use `DefaultNullLogger` instead of `NullLogger{}`. 

Could we achieve something similar without breaking users by leveraging the `NullLogger{}` struct instead?

Also, given that `NullLogger{}` is not a pointer, doesn't sync.Once get copied?